### ISSUE TITLE
TypeError: (msgRollup.value.content.recps || []).map is not a function

### ISSUE THREAD

**Issue by christianbundy** [ssbc/patchbay#274]
Just got this in the latest master. Unclear on how to reproduce, but I wanted to log it in case others got the same bug.

```
TypeError: (msgRollup.value.content.recps || []).map is not a function
    at Object.inboxLayout (/home/christianbundy/src/ssbc/patchbay/node_modules/patch-inbox/plugs/message/html/layout/inbox.js:32:8)
    at Object.layout (/home/christianbundy/src/ssbc/patchbay/node_modules/depject/apply.js:18:30)
    at Object.renderMessage (/home/christianbundy/src/ssbc/patchbay/node_modules/patch-inbox/plugs/message/html/render/post.js:16:36)
    at Object.render (/home/christianbundy/src/ssbc/patchbay/node_modules/depject/apply.js:18:30)
    at render (/home/christianbundy/src/ssbc/patchbay/node_modules/patch-inbox/plugs/post/page/inbox.js:81:31)
    at add (/home/christianbundy/src/ssbc/patchbay/node_modules/pull-scroll/index.js:42:15)
    at /home/christianbundy/src/ssbc/patchbay/node_modules/pull-scroll/index.js:71:11
    at /home/christianbundy/src/ssbc/patchbay/node_modules/pull-stream/sinks/drain.js:24:37
    at /home/christianbundy/src/ssbc/patchbay/node_modules/pull-stream/throughs/async-map.js:39:20
    at onceTrue (/home/christianbundy/src/ssbc/patchbay/node_modules/patchcore/feed/pull/with-replies.js:24:9)
    at Array.watch (/home/christianbundy/src/ssbc/patchbay/node_modules/mutant/once-true.js:8:7)
    at Function.Observable.observable.set (/home/christianbundy/src/ssbc/patchbay/node_modules/mutant/value.js:12:25)
    at pull.drain (/home/christianbundy/src/ssbc/patchbay/node_modules/patchcore/backlinks/obs.js:63:20)
    at /home/christianbundy/src/ssbc/patchbay/node_modules/pull-stream/sinks/drain.js:24:37
    at /home/christianbundy/src/ssbc/patchbay/node_modules/pull-abortable/index.js:46:11
    at PacketStreamSubstream.weird.read (/home/christianbundy/src/ssbc/patchbay/node_modules/muxrpc/pull-weird.js:33:7)
```

**Comment by mixmix** [comment]
I've not seen this but that rollup needs ripping out and re-writing in a way which is optimised for this context. sigh

I did change the stream which was delivering content so that's probably what introduced this.
This is the code I changed 

https://github.com/mixmix/patch-inbox/blob/master/plugs/post/page/inbox.js#L98-L120

I think the problem is likely that `$truthy` operator letting some things through which are not truthy in the way I thought ... yeah it's using `!!recps` which... hmm what the crap is coming through.

sigh, I think we need a `pull.filter(m => Array.isArray(m.value.content.recps))` ?
